corehq/apps/saved_reports/tests/test_models.py

```diff
@@ -32,3 +36,134 @@ def _create_user(self, id='100', email='not-included', is_domain_admin=False):
         user.get_email = lambda: email
 
         return user
+
+
+class TestRecipientsByLanguage(SimpleTestCase):
+    def test_existing_user_with_no_language_gets_default_language(self):
+        report = self._create_report_for_emails('test@dimagi.com')
+        self._establish_user_languages([{'username': 'test@user.com', 'language': None}])
+
+        recipients_by_language = report.recipients_by_language
+        self.assertSetEqual(set(recipients_by_language.keys()), {'en'})
+        self.assertSetEqual(set(recipients_by_language['en']), {'test@dimagi.com'})
```

Comment:
nit: Would a single assertion work here?
```suggestion
        self.assertEqual(recipients_by_language, {'en': ['test@dimagi.com']})
```
(same in the next test below)